tests/ha/ntp_client.pm

```diff
@@ -0,0 +1,24 @@
+use base "hacluster";
+use testapi;
+
+sub run() {
+   script_run("yast2 ntp-client");
+   assert_screen "yast2-ntp-client", 30;
+   send_key 'alt-b'; #start ntp daemon on Boot
+   send_key 'alt-a'; #add new Server
+   assert_screen "yast2-ntp-client-add-source", 30;
+   send_key 'alt-n'; #Next
+   assert_screen "yast2-ntp-client-add-server", 30;
+   type_string "ntp";
```

Comment:
is this meant to be a variable? I suspect this test is a starting point getting ready to be expanded for multi-machine testing?